### ISSUE TITLE
cc.close() function in finally block

### ISSUE THREAD

**Issue by Mengflz** [enarjord/passivbot#405]
When I use the forager.py, I find the script will end after running it for an hour.
I checked the code. It seems the cc.close() is put in finally block. So the instance will be closed in the second time unless the output of the first run. 
I think may be you could think of move the cc instance into the loop? Or move the cc.close() to the except block. But in the later situation, I found it need some extra ways to catch the KeyBoardInterrupt Exception.

**Comment by enarjord** [comment]
Good catch sir.  You are right on every account.  Just pushed a commit where the ccxt instance is moved to inside the loop.
